src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationCorrelationAnalysisPresenter.cs

```diff
@@ -24,7 +24,7 @@ public class ParameterIdentificationCorrelationAnalysisPresenter : ParameterIden
 
       protected override Matrix CalculateMatrix()
       {
-         if (SelectedRunResults.JacobianMatrix == null)
+         if (SelectedRunResults?.JacobianMatrix == null)
```

Comment:
Also fixed this other issue that was preventing loading from a PI without result (from Snapshot)